index.html

```diff
@@ -86,6 +86,192 @@ <h2 class="description">I’m a Software Developer</h2>
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </li>
+        <li class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </li>
+        <li class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </li>
+        <li class="card">
+            <h2 class="card-header">Profesional Art Printing Data</h2>
+            <p class="card-body">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+                <li class="lang">
+                    <a href="#">CSS</a>
+                </li>
+                <li class="lang">
+                    <a href="#">bootstrap</a>
+                </li>
+                <li class="lang">
+                    <a href="#">Ruby</a>
+                </li>
+            </ul>
+            <a href="#" class="see-project">See Project</a>
+        </li>
+    </ul>
+    <section class="about-me-main">
+        <h2 class="about-me-title">About me</h2>
+        <p class="about-me-text">
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#" class="get-resume">Get My Resume</a>
```

Comment:
- [ ] This section is differ in terms of design and also not look great. Please make it similar as per the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

### Your design
![Screenshot from 2021-06-10 17-48-49](https://user-images.githubusercontent.com/22434660/121524447-f5631480-ca14-11eb-9e29-8256641742f6.png)

### Figma design

![Screenshot from 2021-06-10 17-49-15](https://user-images.githubusercontent.com/22434660/121524536-090e7b00-ca15-11eb-91c9-198caaab091d.png)